### ISSUE TITLE
Cannot create new PlatformIO project in CLion

### ISSUE THREAD

**Issue by drewmutt** [platformio/platformio-core#4751]
### Configuration

**Operating system**:
Mac (M1) 12.5.1

**PlatformIO Version** (`platformio --version`):
6.1.11

**Plugin Version**:
233.6745.300 (had same issue in prev versions)

**CLion Version**:
CLion 2023.3 EAP (same issue in 2023.2.2)

### Description of problem
The first step of the create new PlatformIO project, fails. Preventing me from creating new projects in CLion.

#### Steps to Reproduce
1. File > New Project > PlatformIO

### Actual Results
Behold the text "Gathering info..." in the board list, and the following error below it:
`Use JsonReaderEx.setLenient(true) to accept malformed JSON at line 1 column 2`

The "Create" button is expectingly gray, and progress cannot continue.

### Expected Results
Display the list of boards, and allow the user to select one to continue.

### Additional info
Looking into the [source](https://github.com/JetBrains/intellij-plugins/blob/master/platformio/src/com/jetbrains/cidr/cpp/embedded/platformio/project/PlatformioProjectSettingsStep.kt), I found a call to `pio boards --json-output`.

Executing it appears to work correctly, and the JSON doesn't appear to be corrupt.

**Comment by ak1987** [comment]
If your any CLI input (like `pio --version`) comes with warning like this:
`.platformio/penv/lib/python3.9/site-packages/urllib3/__init__.py:34: NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020`
You should try downgrade library with following command:
```
. ~/.platformio/penv/bin/activate
pip install urllib3==1.26.15
```
JFYI: [https://github.com/platformio/platformio-core/issues/4614](url)
_TLDR: Warning comes in board list output causing parser fail_

**Comment by ivankravets** [comment]
@ak1987 , thanks for the solution. It the right solution to check `pio boards` command.

**Comment by Quantriple** [comment]
when i run "pio --version", show  "PlatformIO Core, version 6.1.11", but  clion has been staying gathering info.......,could you please help me ......

**Comment by Bugrabugra** [comment]
> If your any CLI input (like `pio --version`) comes with warning like this: `.platformio/penv/lib/python3.9/site-packages/urllib3/__init__.py:34: NotOpenSSLWarning: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'LibreSSL 2.8.3'. See: https://github.com/urllib3/urllib3/issues/3020` You should try downgrade library with following command:
> 
> ```
> . ~/.platformio/penv/bin/activate
> pip install urllib3==1.26.15
> ```
> 
> JFYI: [#4614](https://github.com/platformio/platformio-core/issues/4614) _TLDR: Warning comes in board list output causing parser fail_

This solved my problematic "pio --version" result but my CLion keeps gathering info

Edit: I have solved my CLion problem as well. when I check the system version of Platform.io with`pio system info`, it showed my Python executable as a conda env. So, downgrading urllib in  ~/.platformio/penv/bin/activate env was not effecting my CLion side at all. So I uninstalled my platformio from conda env by `pip uninstall platformio` then I installed like the recommended installation:

```
curl -fsSL -o get-platformio.py https://raw.githubusercontent.com/platformio/platformio-core-installer/master/get-platformio.py
python3 get-platformio.py
```
It installed brand new platformio env to a folder under my Users folder like 

`/Users/USERNAME/.platformio/penv/bin/python`

and finally I have done the @ak1987's solution and my problem is fully solved.
